src/ProjectImage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Image } from '@chakra-ui/react';
+
+function ProjectImage({ project }) {
+  const [imgUrl, setImgUrl] = React.useState(
+    'https://via.placeholder.com/250?text=SpasiSofia'
+  );
+  Promise.resolve(project.imageUrl).then((url) => {
```

Comment:
issue: this is not really working anymore 

See
https://projects.spasisofia.org/07_%D0%9F%D0%BB%D0%BE%D1%89%D0%B0%D0%B4%D0%BA%D0%B8-%D0%93%D0%B5%D0%BE-%D0%9C%D0%B8%D0%BB%D0%B5%D0%B2.png
<img width="1344" alt="image" src="https://github.com/SpasiSofia/projects-map/assets/483410/50b6cbae-d0f5-4289-9a49-73f41d648c07">

Reply:
Is it not because the image is missing? That was existing code so just moved it around. Do we have correct link for images and/or if this one specific is missing how we can fix it?

Reply:
FIY. Checked few other projects again and their images load properly (where such are provided). I believe the issue here is with the data we have or it is provided so this can be fixed from the source which I believe is the website.

Reply:
The idea here was when the image is not there (not loading) to provide fallback : placeholder image (the default set in the imgUrl variable)
@Thusnake  any thoughts ?

Reply:
This project is from the static ones. The issue was present prior to this PR.
The culprit is this faulty URI in `static-projects.js`: https://github.com/SpasiSofia/projects-map/blob/master/src/data/static-projects.js#L1834

Reply:
Just checked it and sadly this request returns 200 and everything okay so there is no way for me to verify that there is no image behind it ( easily). I think it will be just easier to fix the URL  instead of adding more complexity where we just should provide correct data when filling these details.

Reply:
Do we agree to merge this and sort it out separately? If yes please answer here or just merge the PR. Everything else I believe is sorted.